torch/csrc/jit/codegen/cuda/utils.h

```diff
@@ -97,10 +97,11 @@ TORCH_CUDA_CU_API bool isOptionDisabled(DisableOption option);
 //!
 enum class EnableOption {
   Complex, //! Enable complex support on python
-  KernelProfile, //! Enable intra-kernel performance profiling
-  LinearDecomposition, //! Enable linear-bias decomposition
   ConvDecomposition, //! Enable conv-bias decomposition
   GraphOp, //! Enable graphOps(index_select/gather/scatter)
+  KernelDb, //! Enable Kernel Database
```

Comment:
This probably should be disabled for testing, right?

Reply:
Could you clarify what you mean? By default, the KernelDb is turned off, so it shouldn't impact any other testing.  The KernelDb is specifically enabled for its own testing.